Source/Diagnostics/BTDiagnostics.H

```diff
@@ -0,0 +1,152 @@
+#ifndef WARPX_BTDIAGNOSTICS_H_
+#define WARPX_BTDIAGNOSTICS_H_
+
+#include "Diagnostics.H"
+using namespace amrex::literals;
```

Comment:
We don't want to have any `using namespace` in a .H file. I think the reason you add it here is for the `_rt`, but this is mostly important inside kernels, I don't think you need it here. If you really want to keep it, I would suggest putting the body of function `dz_lab` in the cpp file instead, as it is the only one where you need `_rt`.
```suggestion
```